cli/internal/run/dry_run.go

```diff
@@ -282,30 +289,32 @@ func commandLooksLikeTurbo(command string) bool {
 
 // TODO: put this somewhere else
 type taskSummary struct {
-	TaskID          string           `json:"taskId"`
-	Task            string           `json:"task"`
-	Package         string           `json:"package"`
-	Hash            string           `json:"hash"`
-	CacheState      cache.ItemStatus `json:"cacheState"`
-	Command         string           `json:"command"`
-	Outputs         []string         `json:"outputs"`
-	ExcludedOutputs []string         `json:"excludedOutputs"`
-	LogFile         string           `json:"logFile"`
-	Dir             string           `json:"directory"`
-	Dependencies    []string         `json:"dependencies"`
-	Dependents      []string         `json:"dependents"`
+	TaskID                 string             `json:"taskId"`
+	Task                   string             `json:"task"`
+	Package                string             `json:"package"`
+	Hash                   string             `json:"hash"`
+	CacheState             cache.ItemStatus   `json:"cacheState"`
+	Command                string             `json:"command"`
+	Outputs                []string           `json:"outputs"`
+	ExcludedOutputs        []string           `json:"excludedOutputs"`
+	LogFile                string             `json:"logFile"`
+	Dir                    string             `json:"directory"`
+	Dependencies           []string           `json:"dependencies"`
+	Dependents             []string           `json:"dependents"`
+	ResolvedTaskDefinition *fs.TaskDefinition `json:"resolvedTaskDefinition"`
```

Comment:
Need to bikeshed the `ResolvedTaskDefinition` key name a bit. 

My main concern is that "resolved" is misleading because we aren't going to include things like turbo's injected env vars or expand globs in inputs/outputs. I think that is ok, since this is a "task definition", and that "fully resolved state" is more for "cache hit status".

Reply:
IMO I think the name pretty clear